renovate.json5

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": ["config:base"],
+  "packageRules": [
+    {
+      "matchManagers": ["gomod"],
+      "postUpdateOptions": ["gomodTidy"],
+    },
+    {
+      "matchManagers": ["gomod"],
+      "matchUpdateTypes": ["minor", "patch", "pin"],
+      "automerge": true,
```

Comment:
are all the dependencies semver compatible?

Reply:
@suzuki-shunsuke wdty? I'd say we keep automerge `false` to keep control in the beginning

Reply:
Sorry for late reply.
I think if a pull request includes incompatible changes, the pull request wouldn't be merged automatically because CI would fail.

Reply:
I concern it is hard to check and merge all pull requests manually.

Reply:
In case of major update, we have to update the import path manually.

Reply:
alright